### PR TITLE
fix static config file in tests

### PR DESCRIPTION
**Pull request by adiux**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | maybe
| New feature/enhancement? (use the a.x branch)      | no
| Deprecations?                          | no
| BC breaks? (use the c.x branch)        | no
| Automated tests included?              | yes
| Issue(s) addressed                     | Fixes an issue with automated tests if the config file is not in the default location

#### Steps to test this PR:

1. Checkout the code locally 
2. Run automated tests for this
  - `composer test -- --filter ConfigControllerFunctionalTest` or
  - `ddev exec composer test -- --filter ConfigControllerFunctionalTest`

(I guess the automated tests that run with each PR should also show if there is a problem with this)

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10810"></a>

### CODE REVIEW

File: app/bundles/ConfigBundle/Tests/Controller/ConfigControllerFunctionalTest.php [hunk 1]
```diff
@@ -127,7 +128,7 @@ public function testValuesAreEscapedProperly(): void
 
     private function getConfigPath(): string
     {
-        return self::$container->getParameter('kernel.project_dir').'/app/config/local.php';
+        return ParameterLoader::getLocalConfigFile(self::$container->getParameter('kernel.root_dir'));
```
Comment (fedys) [inline app/bundles/ConfigBundle/Tests/Controller/ConfigControllerFunctionalTest.php]:
```suggestion
        return self::$container->get('kernel')->getLocalConfigFile();
```

File: app/bundles/ConfigBundle/Tests/Controller/ConfigControllerFunctionalTest.php [hunk 2]
```diff
@@ -14,6 +14,7 @@
 namespace Mautic\ConfigBundle\Tests\Controller;
 
 use DateTime;
+use Mautic\CoreBundle\Loader\ParameterLoader;
```
Comment (fedys) [inline app/bundles/ConfigBundle/Tests/Controller/ConfigControllerFunctionalTest.php]:
```suggestion
```

**Review by fedys — CHANGES_REQUESTED** [review]
Thanks @adiux! There is a way to make it a bit simpler. See my code suggestions.

**Review by fedys — APPROVED** [review]
Thanks @adiux 👍

**Review by kuzmany — APPROVED** [review]
👍

**Pull request merged**